File: Assignments/Assignment_15/p3.c
#include<stdio.h>
void main(){

        int row;
        printf("Enter the no of rows : \n");
        scanf("%d",&row);
        int space=row-1,alp=64+row;
        for(int i=1;i<=row;i++){

                alp=64+row-i+1;
                for(int j=1;j<=row;j++){

                        if(j<=space){

                                printf("\t");
                        }else{

                                printf("%c\t",alp);
                                alp++;
                        }
                }space--;
                printf("\n\n");
        }
}

File: JAVA/13.Pyramid_and_Diamond/p3.java
import java.util.*;

class Demo{

	public static void main(String[] args){
	
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter no. of rows:");
		int row=sc.nextInt();
		int space,col;
		char ch1,ch2;

		for(int i=1;i<=row;i++){
		
			ch1='A';
			ch2='a';
			space=row-i;
			col=(i*2)-1;
			for(int sp1=1;sp1<=space;sp1++){
			
				System.out.print("\t");
			}
			for(int j=1;j<=col;j++){
			
				if(i%2!=0){
				
					System.out.print(ch1+"\t");
					ch1++;
				}else{
				
					System.out.print(ch2+"\t");
					ch2++;
				}
			}
			for(int sp2=1;sp2<=space;sp2++){
			
				System.out.print("\t");
			}
			System.out.println();
		}
	}
}
